### ISSUE TITLE
Displacements Breaking Explicit Symmetry Crash Psi

### ISSUE THREAD

**Issue by JonathonMisiewicz** [psi4/psi4#1052]
If a user sets the symmetry explicitly and then attempts to perform a finite difference calculation, Psi raises `Fatal Error: User specified point group is not a subgroup of the highest detected point group.` This problem does not occur if the user does not explicitly provide the symmetry. The problem also occurs if `reset_point_group` is used. Example input below.

I don't know enough about Psi's symmetry and molecule tech to tell if this is a problem that needs to be fixed findif side or molecule side. If this needs to be fixed findif side, I'll take it for obvious reasons. If not, then if this isn't fixed by the time findif is done, I'll take this anyways. I've had a vendetta against this bug since November.

```
molecule h2o {
  symmetry c2v
  O
  H 1 0.9894093
  H 1 0.9894093 2 100.02688
}

set {
  basis sto-3g
  d_convergence 11
  scf_type pk
}

frequencies('scf', dertype=1, return_wfn=True)
```

**Comment by dgasmith** [comment]
@JonathonMisiewicz I believe you fixed this in a PR or am I wrong.

**Comment by JonathonMisiewicz** [comment]
@dgasmith I never did. I was playing with fixes, but those got stymied by direction dependent subgroups. Still, this is item number three on my hit list of bugs, and by far easier to fix than items one and two. I have a paper to prioritize, but I can hopefully include this as a bug fix between the feature freeze and 1.3 release.
